reducers/views/search.js

```diff
@@ -15,6 +15,17 @@ function modalSearch(state = '', action) {
     }
 }
 
+function systemUsersSearch(state = {}, action) {
```

Comment:
It might be better to have this under `reducers/views/admin.js` unless we want open this page to more people

Reply:
Meh, I think it doesn't matter that much but if you're more than 0/5 I'll change it